### DISCUSSION TITLE
Saturable plasma protein binding model

### DISCUSSION THREAD

**Discussion by AlejandraSchiavo** [Open-Systems-Pharmacology/Forum#907] [Models] [answered]
Hello everyone,

We are working in a PBPK model for a drug which has saturable plasma protein binding kinetic. As strategy, we tried to model free plasma concentrations using unbound fraction (fu) equal to 1 (no real value) and free plasma clearance, and then calculate the total plasma concentrations. However, we have this question: Is fu used for something else besides estimate free plasma concentrations? We read it is used to estimate partition distributions. If the answer is "yes", what strategy do you recommend to model  a saturable plasma protein binding process?
Thank you, 
Best regards,

Alejandra

**Comment by StephanSchaller** [comment]
> As strategy, we tried to model free plasma concentrations using unbound fraction (fu) equal to 1 (no real value) and free plasma clearance, and then calculate the total plasma concentrations

I am not sure I can follow what you have done, but to model concentration-dependent fu, you would need to go to MoBi and change the value of fu to a formula.
Fu is considered for the calculation of partition coefficients but this can remain as only the unbound fraction is subject to distribution.

In the past, we have modeled concentration-dependent fu (for a compound primarily bound to albumin) by explicitly modeling albumin as a compound and implementing an albumin binding process in MoBi.

hope this helps

**Reply by AlejandraSchiavo** [reply]
Hi Stephan,

We tried to change the value of fu in MoBi, but we could not change it to a formula. Anyway, we will try again. Your strategy seems interesting, maybe it could be useful, due to our drug primarily bound to albumin too. 
Thanks for your help.

**Reply by SierraCoonts** [reply]
Hi @StephanSchaller, 

I am in the process of developing a model for a drug that will need fu to be modeled in a concentration dependent manner. Could you provide some resources on how to implement the albumin binding process in MoBi? Will modeling albumin as a compound be different than how it would be modeled as a protein binding partner?

Thank you,

Sierra

**Comment by ekaratza** [comment, marked as answer]
Hi Alejandra, 

I had a similar a similar case with a drug that had saturable plasma protein binding kinetics. 
An accurate value of fraction unbound at each time point is very important as it significantly affects clearance and volume of distribution. In PKSim/Mobi suite you can do that very efficiently. The idea is to implement in PKsim/Mobi an equation that relates the fu as a dependent variable to the total concentration of your drug that will be the independent variable . 

So in order to implement in your model the change of fraction unbound in relation to your drug's concentration you will have to : 
1. Find in the literature the fraction unbound baseline (Fubase)  , i.e. the lowest fraction unbound noted for your compound and use it in your compound -> physicochemical parameters block 
2. Identify in the literature or develop through regression an equation linking the fraction unbound (FU) to the total drug concentration.
3. Perform the regression and/or modifications to the equation from step 2 to retrieve an equation linking the FU/Fubase to the total drug concentration 
4. Export your simulation in Mobi
5. In Mobi: Go to the building block ‘Spatial Structures’  -> Tree  -> MoleculeProperties and choose the folder parameters. 
6. There add the total venous concentration  of your drug as a new parameter in the following way: 
a)	Click ‘add parameter’ of type: Global, Dimension: Concentration (mass or molar depending on your equation) , Group: Mobi, Formula type: Formula (an explicit formula) and name it eg. Ctotal 
b)	Then right click in the white space prepared for adding equation parameters and ‘add reference’.
c)	There include the total venous blood concentration using the path: Organism|VenousBlood|Plasma|MOLECULE|Concentration
d)	If your equation was developed with the  total drug concentration in molar units you can continue to the next steps, if it was developed with the  total drug concentration in mass units then add also the include the molecular weight. 
e)	Set the dimensions of this parameter in the table carefully to much the dimensions of your fu~Ctotal equation. 

You might end up having something like this: 
![image](https://user-images.githubusercontent.com/85250758/120673233-fdddad00-c460-11eb-8577-4bcb0d50fa31.png)

7. Add a new parameter of type: Global, Dimension: Dimensionless , Group: Mobi, Formula type: Formula (an explicit formula) and name it eg. Dynamic . Then add you total concentration as a parameter of the equation for the quantification of this parameter and then write the equation you obtained in step 3 above. 
You might end up having something like this: 


Please feel free to contact me if you need additional information.
![image](https://user-images.githubusercontent.com/85250758/120673322-1057e680-c461-11eb-92e8-7ff7d97ae122.png)

8. Go to ‘Spatial Structures’  -> Tree  -> MoleculeProperties -> Parameters -> Fraction unbound (plasma) and multiply the parameter you have created on step 7 with the equation  already set there for the estimation of the Fraction unbound 
Like this: 
![image](https://user-images.githubusercontent.com/85250758/120673376-1c43a880-c461-11eb-9f08-ba7593b1dd86.png)

9. Update your simulation with the new version of your spatial structure and then run your simulation. 



Hope that it helps, 
Eleni 

Please find an example of the 'spatial structure' after doing the process above here: 
[scalar_fu.zip](https://github.com/Open-Systems-Pharmacology/Forum/files/6593120/scalar_fu.zip)

**Reply by nina-nauwelaerts** [reply]
Hi Eleni

I noticed you use a function for Fu/Fubase. I was wondering why you would not do the regression immediately for Fu related to total concentration? Is there any reason why you also need to include the Fubase? 

Could you please explain why you need the Ctotal? What does this max(Ctotal; 0) function do? Is Organism|VenousBlood|Plasma|MOLECULE|Concentration not already the total plasma concentration? 

Kind regards
Nina

**Reply by wink02** [reply]
I'm very sorry. This is my first time using the Mobi module. I have a basic question. Before creating the "fu" formula for the drug using Mobi, do I need to first complete the basic settings for the ADME (absorption, distribution, metabolism, excretion) of the drug in Pksim, and then complete the model fitting in the Mobi section after adding the fu? Thank you for your reply.

**Comment by AlejandraSchiavo** [comment]
Hi Eleni,
Thank you very much for your step by step guide, it will useful to us since we could not change the fu as a dependent variable before that. We will read it and implement this strategy in PK-Sim/MoBi.

**Comment by ekaratza** [comment]
Hi Nina,

Thank you for reaching out.

Please find the answers below :

I noticed you use a function for Fu/Fubase. I was wondering why you would not do the regression immediately for Fu related to total concentration? Is there any reason why you also need to include the Fubase?

It is mandatory to set a reference fraction unbound value in the compound block that will be used when the concentration is 0 as a starting point.

The equation that is developed need to have as dependent variable the Fu corrected with the baseline (Fu/Fubaseline) as this equation is going to be used as a scalar to multiply with the baseline fraction unbound value within the block 'spatial structures' .

So in the spatial structure before including your scalar you have this pre-specified equation for fu :
1 / (1 + (BindingPartner = Albumin ? AlbuminOntgenyFactor : (BindingPartner = Agp ? AgpOntogenyFactor : 1) ) * (1 - fu_ref) / fu_ref) * PlasmaProteinScaleFactor

 which for normal adults equalts to fu reference value, i.e. the baseline value set in the compound block in pksim (it was expressed this way to allow the flexibility to scale in children or to pathological states) .

If you  develop your scalar equation as fu then it will turn out that the resulting fu, used in the simulation will be
Fu reference * Fu (the fu value yielded for your equation) , on the contrary if you develop and you scalar equation for fu/fu reference then when you implent it you will have Fu reference* Fu/Fu reference =Fu (the value of fraction unbound that you want to use for each concentration).


Could you please explain why you need the Ctotal?

You need to define as a new parameter the Concentration of your drug in the venous blood and then use this parameter as part of your equation that estimates the fu as a function of concentration.


What does this max(Ctotal; 0) function do?

It prevents the ODE solver to yield negative values for Ctotal (can happen for very low concentrations) that will not allow your simulations to run. It is a protection that says that Ctotal is allowed to range from 0 and above.



Is Organism|VenousBlood|Plasma|MOLECULE|Concentration not already the total plasma concentration?

Yes it is but adding it as a parameter allows you to set the dimensions that you want, i..e mg/L or mM or umol/L etc and allows you to introduce protection as described before .

I hope that it helps.

Best regards,
Eleni


________________________________
Από: nina-nauwelaerts ***@***.***>
Στάλθηκε: Πέμπτη, 17 Ιουνίου 2021 2:03 μμ
Προς: Open-Systems-Pharmacology/Forum ***@***.***>
Κοιν.: ekaratza ***@***.***>; Comment ***@***.***>
Θέμα: Re: [Open-Systems-Pharmacology/Forum] Saturable plasma protein binding model (#907)


Hi Eleni

I noticed you use a function for Fu/Fubase. I was wondering why you would not do the regression immediately for Fu related to total concentration? Is there any reason why you also need to include the Fubase?

Could you please explain why you need the Ctotal? What does this max(Ctotal; 0) function do? Is Organism|VenousBlood|Plasma|MOLECULE|Concentration not already the total plasma concentration?

Kind regards
Nina

**Reply by AlejandraSchiavo** [reply]
Hi Eleni,
How are you?
I am using your step by step guide, it is wonderfull!
I have this question about the parameter "Ctotal": What are the concentration (molar and mass) units?
I am not sure, and I can not write my fu vs Ctotal equation because of that.

Thank you for your help

Best regards,
Alejandra

**Reply by ekaratza** [reply]
Hi Alejandra, 

In my case it was concentration (mass) units and i have set the value of Ctotal to be in mg/L. 
However, your Ctotal parameter could be in any dimensions of concentration that you need based on the way that your equation was developed. 

Therefore, if your equation was developed using drug concentrations in mmol/L then you should set your Dimensions as Concentration (mass) and do not correct per MW. 

Hope that it helps. 

Best regards, 
Eleni

**Reply by msevestre** [reply]
@AlejandraSchiavo Just keep in mind that whatever dimension you are using, your rates need to be consistent with the expected dimension of the system. You cannot add mass and molar concentration for example. Internally, concentrations are saved in umol/l. So if you perform some operation on concentration variables,  please make sure that you verify the output to make sure they make sense

**Reply by AlejandraSchiavo** [reply]
Hi Eleni and Michael,
Thank you so much for your help!
I could do it,

Best regards,
Alejandra

**Comment by ekaratza** [comment]
There is such an example here:
<https://github.com/Open-Systems-Pharmacology/Valproic-Acid>
Open-Systems-Pharmacology/Valproic-Acid: Whole-body PBPK model of valproic acid including non-linear binding with albumin in adults, children and children with hypoalbuminemia.<https://github.com/Open-Systems-Pharmacology/Valproic-Acid>
github.com<https://github.com/Open-Systems-Pharmacology/Valproic-Acid>
[apple-touch-icon-180x180-a80b8e11abe2.png]<https://github.com/Open-Systems-Pharmacology/Valproic-Acid>

That might help.
Best regards,
Eleni

On Jun 26, 2025, at 5:37 PM, SierraCoonts ***@***.***> wrote:

﻿

Hi @StephanSchaller<https://github.com/StephanSchaller>,

I am in the process of developing a model for a drug that will need fu to be modeled in a concentration dependent manner. Could you provide some resources on how to implement the albumin binding process in MoBi? Will modeling albumin as a compound be different than how it would be modeled as a protein binding partner?

Thank you,

Sierra

**Reply by SierraCoonts** [reply]
Hi Eleni, 

Thank you for the example! This is very helpful.

-Sierra
